packages/emotion/typings/emotion.d.ts

```diff
@@ -0,0 +1,47 @@
+export type Interpolation = string | number | boolean | null | undefined | _Interpolation1 | _Interpolation2 | _Interpolation3;
+
+// HACK: See https://github.com/Microsoft/TypeScript/issues/3496#issuecomment-128553540
+interface _Interpolation1 extends Record<string, Interpolation> {}
+interface _Interpolation2 extends Array<Interpolation> {}
+interface _Interpolation3 {
+  (): Interpolation;
+}
+
+export type CreateStyles<TRet> = ((value: Interpolation) => TRet)
```

Comment:
I'm not familiar with TypeScript but I think this means that it can only have something as the first argument. `createStyles` accepts an arbitrary length of arguments, so maybe something like this?

```typescript
(...values: Interpolation[])
```

Reply:
Fixed.